platform/security/sts/security-sts-propertyclaimshandler/pom.xml

```diff
@@ -74,6 +74,11 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
+            <groupId>io.fastjson</groupId>
+            <artifactId>boon</artifactId>
+            <version>0.33</version>
```

Comment:
We're using boon in a few other places as well, at version 0.30. We should probably create a property and normalize on this version if it's the most stable.